include/miopen/miopen.h

```diff
@@ -1730,6 +1730,14 @@ miopenFindConvolutionForwardAlgorithm(miopenHandle_t handle,
  * The scaling parameter alpha (float) and shift parameter beta (float) are only supported for
```

Comment:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/8be2998d7a1210b7bc82d778f7df0d92aa65b57d/include/miopen/miopen.h#L636

Mention that it creates _packed_ tensor with NCHW layout.